Update docs to address the CEIP prompt in automation (#216)
vmware-tanzu/tanzu-cli: docs/quickstart/install.md

sudo yum install tanzu-cli # dnf install can also be used
```

## Continuous Integration

****Customer Experience Improvement Program (CEIP) Prompt**** :

> CLI would prompt users for CEIP participation on their first tanzu command usage except `tanzu version`

Reviewer: ```suggestion
The Tanzu CLI prompts the user for CEIP participation on their first use of any `tanzu` command
```

Author: Updated. Thanks @marckhouzam .

> , `tanzu ceip-participation set` and `tanzu config set` commands.

Reviewer: ```suggestion
except for `tanzu version`, `tanzu ceip-participation set` and `tanzu config set`.
```

Author: updated.

> Users installing/using the CLI through CI or by any automation means, should be able to skip the prompt by

Reviewer: ```suggestion
Users installing/using the CLI through CI or by any automation means should be able to skip the prompt by
```

Author: Updated
